src/app/chat/chat-room/chat-room.component.html

```diff
@@ -137,9 +127,44 @@
           (ngModelChange)="typing()"
           appAutoresize="100"
           rows="1"
+          (ionFocus)="textFieldFocused = true;"
         ></ion-textarea>
       </ion-col>
-      <ion-col size="1" class="ion-no-padding ion-text-center">
+      <ng-container *ngIf="utils.isMobile() && !textFieldFocused">
+        <ion-col class="ion-no-padding ion-text-center">
+          <ion-button (click)="attach('video')" fill="clear" class="ion-no-padding">
+            <ion-icon name="film-outline" slot="icon-only"></ion-icon>
+          </ion-button>
+        </ion-col>
+        <ion-col class="ion-no-padding ion-text-center">
+          <ion-button (click)="attach('image')" fill="clear" class="ion-no-padding">
+            <ion-icon name="images-outline" slot="icon-only"></ion-icon>
+          </ion-button>
+        </ion-col>
+        <ion-col class="ion-no-padding ion-text-center">
+          <ion-button (click)="attach('any')" fill="clear" class="ion-no-padding">
+            <ion-icon name="attach-outline" slot="icon-only"></ion-icon>
+          </ion-button>
+        </ion-col>
```

Comment:
Try to reuse this block of HTML with `ng-template` so that we are not duplicating codes